### DISCUSSION TITLE
Soil data preparation issues

### DISCUSSION THREAD

**Discussion by suyx0229** [iiasa/CWatM#135] [Q&A] [answered]
Dear CWatM community: 
  
I am currently doing cwatm simulation with 1km resolution. When I was preparing the soil data, I found that each soil hydraulic parameter is divided into 1, 2, and 3, such as alpha1, alpha2, alpha3. Please tell me about 1 and 2. ,3 is divided based on what?
![soil问题](https://github.com/iiasa/CWatM/assets/169023319/5347a1be-bf08-4b48-b971-1404625794e9)

**Comment by dof1985** [comment]
Dear @suyx0229 

Welcome to the CWatM user community. Very briefly, the community model soil module simulates soil process for different soil layers (hence 1-3). The thickness of the top-soil (layer 1) is fixed at 5 cm, and the thickness of the two other layers is provided as inputs in the [SOIL] section of the settings files (variables are: StorDepth1, StorDepth2). The  calculation of soil thickness can be found in the 'soil.py' file (search for 'SOIL DEPTH').

All other soil variables represent soil properties for each layer. We recommend to use the data fir SoilGrids from the following dataset: [http://globalchange.bnu.edu.cn/research/soil5.jsp](http://globalchange.bnu.edu.cn/research/soil5.jsp). We have used the following soil horizon to represent CWatM layers: 0 -0.05m (layer 1); 0.15 -0.3m (layer 2); 0.6 -1.0m (layer 3).

Best
Dor

**Reply by suyx0229** [reply]
Thank you very much for answering my confusion, professor, but I found that there is also alpha and forest_alpha in the 30min data preparation. Are the two distinguished by soil coverage? In addition, how are the storageDepth, percolationlmp, and cropgrp files obtained? I searched the CWatM manual and found no relevant instructions.
![微信图片_20240613155654](https://github.com/iiasa/CWatM/assets/169023319/5338505f-e48b-46d8-ad7b-5ae10b89baf5)

**Reply by dof1985** [reply, marked as answer]
Hello again

The forest specific soil parameters is still a gap, and is on our to-do-list. Currently, just use the non-forest soil parameters for the forest landcover as well.

While that data can be constructed from different online available data, the easiest approach would be to resample CWatM 5' dataset to 30'' for your specific casestudy. You can download the 5' dataset here: [https://github.com/iiasa/CWatM-Earth-30min](https://github.com/iiasa/CWatM-Earth-30min); See link for 5' in the description.

Best
Dor

**Reply by suyx0229** [reply]
Thank you very much for answering my question, Professor

Best wishes!
suyanxin
